Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityEyeGazeProvider.cs

```diff
@@ -16,11 +16,21 @@ public interface IMixedRealityEyeGazeProvider : IMixedRealityGazeProvider
         /// </summary>
         bool IsEyeGazeValid { get; }
 
+        /// <summary>
+        /// Whether eye tracking data is available.
+        /// </summary>
+        bool IsEyeTrackingAvailable { get; }
+
```

Comment:
I'm not exactly understanding the difference between IsEyeGazeValid  and IsEyeTrackingAvailable?
IsEyeTrackingAvailable also sounds more like is the capability available compared to is recent data available/valid

Reply:
IsEyeGazeValid needs both eye tracking to be available and eye tracking selected as default gaze source. IsEyeTrackingAvailable is there if user wants to query eye tracking ray, even if head gaze is the default. 

Perhaps IsEyeGazeValid should be called something like IsGazeEyeBased ?

Reply:
> Perhaps IsEyeGazeValid should be called something like IsGazeEyeBased ?

IsGazeEyeBased makes more sense to me, or IsGazeInputEyeBased